test/correctness/simd_op_check.cpp

```diff
@@ -1485,7 +1485,7 @@ int main(int argc, char **argv) {
     }
 
     // Compile a runtime for this target, for use in the static test.
-    compile_standalone_runtime("simd_op_check_runtime.o", target);
+    compile_standalone_runtime(Outputs().object("simd_op_check_runtime"), target);
```

Comment:
sgtm

Might be worth considering whether compile_standalone_runtime should just always output a static library